apps/agreement/contracts/Agreement.sol

```diff
@@ -752,6 +753,17 @@ contract Agreement is IAgreement, AragonApp {
         return _isDisputed(challenge);
     }
 
+    /**
+    * @notice Check if a certain amount for a certain address can be unlocked
+    * @dev It always returns false because if we allow owners to unlock by themselves,
+    *      then the unlock call on closing or settling actions could fail due to
+    *      insufficient balance.
```

Comment:
Just to double check, if the Agreement calls unlock itself, as the manager, this function won't be checked?

Reply:
Yes, that’s correct. As you can see [here](https://github.com/aragon/staking/blob/master/contracts/Staking.sol#L621), there is a shortcut for that case.